perma_web/perma/templates/user_management/create-link.html

```diff
@@ -170,10 +169,24 @@ <h3 class="link-header body-ah">Your Perma Links</h3>
 
       {{#each links}}
         <div class="item-container _isExpandable{{#if is_private }} _isPrivate{{/if}}{{#if is_failed }} _isFailed{{/if}}">
-
           <div class="row item-row row-no-bleed _isDraggable" data-link_id="{{ guid }}">
+            <!-- this input is here only so on details expand or collapse
+            the user doesn't wind up all the way up at the top of the screen again -->
```

Comment:
Hooray for explicitly handling focus! This is indeed super important. 

When are you finding that focus gets reset to the body element (e.g., the "top of the page")? That most commonly results when a) the element that had keyboard focus is zapped from the DOM by JS, or b) when you try to follow a link to an in-page anchor that doesn't exist. For instance, `<a href="#no-element-on-the-page-has-this-id">` or c) when you follow a link whose href is "#" and the js that runs doesn't include `event.preventDefault()`. (In this last case, it's usually a good hint that you are really dealing with a "button" rather than a "link", semantically.

I think what you want to have happen here is have focus toggle between the "Show Link Details" button/link and the "Expand Link Details" button/link, kind of like when you are toggling a dropdown open/closed. For that to work, each of these `<a>` needs a unique id, but since you have `{{ guid }}` sounds like that won't be too tough!